js/index.js

```diff
@@ -1 +1,61 @@
 // Your code goes here
+
+const navLinks = document.querySelectorAll('.nav-link');
+navLinks[0].addEventListener('click', (e) => e.preventDefault())
+navLinks[1].addEventListener('click', (e) => e.preventDefault())
+navLinks[2].addEventListener('click', (e) => e.preventDefault())
+navLinks[3].addEventListener('click', (e) => e.preventDefault())
+
+
+const intro = document.querySelector('.intro');
+intro.addEventListener('mouseenter', (e) => {
+    intro.style.border = '2px solid blue'
+})
+
+intro.addEventListener('mouseleave', () => {
+    intro.style.border = ''
+})
+
+window.addEventListener('scroll', () => {
+    const scrolled = window.scrollY;
+    console.log(scrolled);
+    if (Math.floor(scrolled) === 1016)
+    alert ('You\'ve reached the bottom!')
+})
```

Comment:
I would be a little worried about the exact value, I think if it were me I would try >= 1016 just to be safe!